packages/storybook-utils/package.json

```
@@ -28,16 +28,18 @@
     "test:coverage": "vitest run --coverage"
   },
   "peerDependencies": {
+    "@sit-onyx/shared": "workspace:^",
     "@sit-onyx/icons": "workspace:^",
     "@storybook/vue3": ">= 8.3.0",
-    "sit-onyx": "workspace:^",
     "storybook": ">= 8.3.0",
     "storybook-dark-mode": ">= 4"
   },
   "dependencies": {
     "deepmerge-ts": "^7.1.3"
   },
   "devDependencies": {
-    "vue": "catalog:"
+    "vue": "catalog:",
+    "@sit-onyx/shared": "workspace:^",
+    "@sit-onyx/icons": "workspace:^"
```

```suggestion
    "vue": "catalog:"
```

These two are already peerDependencies so they can be removed here

---

updated turbo to determine dependencies itself

    duplicated peerDependencies from the workspace into devDependencies as turbo will not consider them otherwise (See also: vercel/turborepo@1eb3333)